views/order_view.py

```
@@ -0,0 +1,118 @@
+        # 查询订单
+        orders, total = order_service.get_orders(page, page_size)
+        
+        # 返回成功响应
+        return json({
+            "orders": orders,
+            "total": total,
+            "page": page,
+            "page_size": page_size
+        })
+        
+    except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)